IdentityCore/src/cache/accessor/MSIDLegacyTokenCacheAccessor.m

```diff
@@ -206,6 +215,40 @@ - (MSIDToken *)getATForAccount:(MSIDAccount *)account
                  requestParams:(MSIDRequestParameters *)parameters
                        context:(id<MSIDRequestContext>)context
                          error:(NSError * __autoreleasing *)error
```

Comment:
nit: __autoreleasing has not been seen for other classes yet. I believe this could've come from auto completion, should we remove them for consistancy?

Reply:
Yes, agree